SingularityService/src/main/java/com/hubspot/singularity/smtp/MailTemplateHelpers.java

```diff
@@ -34,6 +37,8 @@
   private static final String REQUEST_LINK_FORMAT = "%s/request/%s";
   private static final String LOG_LINK_FORMAT = "%s/task/%s/tail/%s";
   private static final String DEFAULT_TIMESTAMP_FORMAT = "MMM dd h:mm:ss a zzz";
+  private static final Optional<String> DEFAULT_TASK_LOG_ERROR_REGEX = Optional.of("ERROR");
```

Comment:
Any defaults for configuration items should live in the configuration class (i.e. https://github.com/HubSpot/Singularity/pull/979/files#diff-c39ca42e7478ecb6adcf0bf05ac23373R75)